pkgs/os-specific/darwin/apple-sdk-11.0/default.nix

```
@@ -6,7 +6,7 @@
 }:
 
 let
-  mkStub = callPackage ../apple-sdk/mk-stub.nix { } "11.0";
+  mkStub = callPackage ../apple-sdk/mk-stub.nix { } "apple_sdk_11_0" "11.0";
```

```suggestion
  mkStub = callPackage ../apple-sdk/mk-stub.nix { } "darwin.apple_sdk_11_0" "11.0";
```

No reason not to keep the `darwin.` breadcrumb here, right? It’s the canonical attribute path.